examples/multicsc/plot_sample_evoked_response.py

```diff
@@ -264,6 +264,34 @@
 print("\rDisplayed {} atoms".format(len(plotted_atoms)).rjust(40))
 fig.tight_layout()
 
+###############################################################################
+# Here is a different visualization of the atoms correlated with the event
+# onsets. We first select two atoms based on previous figure. Each event
+# corresponds to a column. Then we pick the channel with the largest absolute
+# value in the spatial map of selected atoms. The top row displays the raw
+# timecourses on this channel, aligned on the event onsets, and ordered by
+# minimum raw value. The second row plots the average evoked timesourse,
+# either on the raw signal or on the signal reconstructed using the two
+# selected atoms. The third raw uses the same epoch order than the first row,
+# but displays the signal reconstructed using the two selected atoms. Colored
+# dots are present where each atom is activated. The last row depicts
+# histograms of activations for the selected atoms.
+from alphacsc.viz.epoch import plot_epochs_of_selected_atoms
+from copy import deepcopy
+
+# Select a channel to plot. None will select the channel automatically.
+channel = None
+
+# select the atoms to plot
+idx_atoms = [6, 30]
+
+this_info = deepcopy(info)
+this_info['event_id'] = [1, 2, 3, 4]
+this_info['events'] = events
+
+plot_epochs_of_selected_atoms(cdl, z_hat, X, this_info, t_lim, idx_atoms,
+                              channel=channel, align=False)
```

Comment:
what are the different colors in the third subplot? do you show both the activations and reconstructed signal?

Reply:
Third row:
- Dots are the activations, same colors as fourth row (the dot is not the activation onset but the time at its maximum absolute value).
- Reds and Blues are the reconstructed signals, same colormap and vmin/vmax as first row.